<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabDiveStatistics</class>
 <widget class="QWidget" name="TabDiveStatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statistics</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>382</width>
        <height>286</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         
         
         <item row="0" column="0">
          <widget class="QLabel" name="yVarLabel">
           <property name="text">
            <string>Vertical axis</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
            <widget class="QComboBox" name="yVarName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>     
           </widget>
          </item>
         <item row="2" column="0">
          <widget class="QLabel" name="spacer1">
           <property name="text">
            <string> </string>
           </property>
          </widget>
         </item>

         <item row="3" column="0">
          <widget class="QLabel" name="xVarLabel">
           <property name="text">
            <string>Horizontal axis</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
           <widget class="QComboBox" name="xVarName">
           </widget>
          </item>
         <item row="5" column="0">
          <widget class="QLabel" name="spacer2">
           <property name="text">
            <string> </string>
           </property>
          </widget>
         </item>


         <item row="6" column="0">
          <widget class="QLabel" name="cCodedLabel">
           <property name="text">
            <string>Coded variable</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
           <widget class="QComboBox" name="cCodedName">
           </widget>
          </item>
         <item row="8" column="0">
          <widget class="QLabel" name="spacer3">
           <property name="text">
            <string> </string>
           </property>
          </widget>
         </item>


         <item row="9" column="0">
          <widget class="QLabel" name="graphTypeLabel">
           <property name="text">
            <string>Graph type</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
           <widget class="QComboBox" name="graphType">
           </widget>
          </item>
         <item row="11" column="0">
          <widget class="QLabel" name="spacer4">
           <property name="text">
            <string> </string>
           </property>
          </widget>
         </item>


         <item row="12" column="0">
          <widget class="QLabel" name="xVarLabel">
           <property name="text">
            <string> </string>
           </property>
          </widget>
         </item>
         
         <item row="13" column="0">
          <widget class="QPushButton" name="createGraphButton">
           <property name="text">
            <string>Create graph</string>
           </property>
          </widget>
         </item>

         <item row="14" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>

         <item row="0" column="1" rowspan="15">
         


           <widget class="QQuickWidget" name="graphImage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>     
          </widget>
         </item>

         
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets/>
 <resources/>
 <connections/>
</ui>
